Designer.xaml.cs

```diff
@@ -197,27 +387,61 @@ private void CreateElementInDesignerFrame(object sender, TappedEventArgs e)
     private void AddDesignerGestureControls(View newElement)
     {
         newElement.PropertyChanged += ElementPropertyChanged;
+
         var tapGestureRecognizer = new TapGestureRecognizer();
         tapGestureRecognizer.Tapped += EnableElementForOperations;
         tapGestureRecognizer.Buttons = ButtonsMask.Primary | ButtonsMask.Secondary;
+
         var rightClickRecognizer = new TapGestureRecognizer();
         rightClickRecognizer.Tapped += ShowContextMenu;
         rightClickRecognizer.Buttons = ButtonsMask.Secondary;
+
+        // Cursor changes to pointer on an Element
+        var pointerGestureRecognizer = new PointerGestureRecognizer();
+        pointerGestureRecognizer.PointerEntered += (s, e) => ChangeCursorToHand(s);
+        pointerGestureRecognizer.PointerExited += (s, e) => ChangeCursorToDefault(s);
+
         newElement.GestureRecognizers.Add(tapGestureRecognizer);
         newElement.GestureRecognizers.Add(rightClickRecognizer);
+        newElement.GestureRecognizers.Add(pointerGestureRecognizer);
+    }
+
+    private void ChangeCursorToHand(object sender)
+    {
+        var view = sender as View;
+        if (view != null)
+        {
+            (view.Handler.PlatformView as Xamls.UIElement).ChangeCursor(Inputs.InputSystemCursor.Create(Inputs.InputSystemCursorShape.Hand));
+        }
+    }
+
+    private void ChangeCursorToDefault(object sender)
+    {
+        var view = sender as View;
+        if (view != null)
+        {
+            (view.Handler.PlatformView as Xamls.UIElement).ChangeCursor(Inputs.InputSystemCursor.Create(Inputs.InputSystemCursorShape.Arrow));
+        }
     }
 
     private void ShowContextMenu(object? sender, TappedEventArgs e)
     {
         var location = e.GetPosition(designerFrame).Value;
-        // set margin of the context menu to current mouse position
-        contextMenu.Margin = new Thickness(location.X, 20);
+        // Set margin of the context menu to current mouse position
+        contextMenu.Margin = new Thickness(location.X, location.Y, 0, 0);
```

Comment:
This was done to prevent the context menu from going out of bounds, but looking back at it, now it seems better to put it as a scroll view instead.